db/migrations/0013_m2_5_upgrades.sql

```diff
@@ -0,0 +1,46 @@
+DROP TABLE provider_urls;
+DROP TABLE indexer_proof_set_rails;
+DROP TABLE indexer_proof_sets;
+DROP TABLE indexer_roots;
+DROP TABLE proof_set_stats;
+DROP TABLE retrieval_logs;
+
+CREATE TABLE providers (
```

Comment:
I would suggest adding a `active` column to the providers table as providers can have `isActive` field inside the struct.